### ISSUE TITLE
Right panel no longer persists in the same way between rooms

### ISSUE THREAD

**Issue by turt2live** [vector-im/element-web#20436]
Previously it'd try and keep the same kind of panel open between rooms, but some recent change has meant it is closed by default and is per-room. This leads to jarring timeline behaviour as the width changes between rooms

**Comment by jryans** [comment]
Yes, it is currently expected to be per-room after implementing the threads / maximised widgets [right panel navigation changes](https://github.com/vector-im/element-web/issues/19660#issuecomment-971574851).

It does have the downside that the width may change now, yes.

**Comment by SimonBrandner** [comment]
Personally, I find it quite annoying to have to open the right panel in every room by hand

**Comment by toger5** [comment]
I was running it for some weeks now and got really used to it. If you have set the right panel in most rooms to what you want it to be the right panel in each room it becomes nicer to use. I can see how the transition is not that pleasant. We could consider a default right panel and an additional global right panel open/close state?


But this is definitely a complex UX decision. I am biased of course. But still I have an insight in the arguments made for that change so I can list them for the discussion:
On the one hand it is really beneficial to be able to have the meter list open in a room where you might want to check the members often (bigger room) or not have any panel in a private chat where the member panel content is obvious. Or pinned messages in a room where they are used for annocements or a chat in a room where there is a maximised widget and nothing where the room is only used for the widget and not really to interact in chat ... (you get the point in some situations it is beneficial to have it per room)

The other side of already got mentioned here: opening the panel for each room is annoying because the default is: no right panel.
And that the state changes per room leading to a different timeline width in different rooms.

With threads and widgets the trend seem to be, that there is more per room information next to the timeline. So having the right panel state as part of the room seems reasonable to me.

A more sophisticated solution could be:
Local right panel phases: timeline & thread & right panel widgets.
Global right panel state: all the others.
We could only store it per room if one of the local phases is open. And otherwise just show the current global phase (and follow the global open state)

**Comment by SimonBrandner** [comment]
For me personally, a default opened state would probably be enough
